### DISCUSSION TITLE
AI inference on Ryzen 7 4700u - Hp pavillon 15-eh0xxx

### DISCUSSION THREAD

**Discussion by delta-whiplash** [DavidS95/Smokeless_UMAF#70] [Q&A]
Hi all, I'm trying to use my old laptop to do AI inference, I have 2x16go 2133mhz stick in it at the moment. 
I would like to know how I can try to overclock it to 3200mhz (maximum frequency for my processor according to the [amd website](https://www.amd.com/fr/products/specifications/processors.html)) and if that fails how I can get back to something working.
I have only set my vram size to 16gb, but I don't know if I can improve my inference speed.

Thanks for any help or suggestions :)

**Comment by Espionage724** [comment]
For a Zen 2 2700X, I ran 4 sticks at 3200MHz but had to increase RAM voltage to `1.4V` (something about more sticks, how they're wired on the motherboard in chain, and Zen 2; X470 board and sticks were rated at least 3600)

I'm thinking any general Zen 2 RAM OC guide would be applicable if you can see the BIOS options to do it (likely AMD CBS). A CMOS reset _should_ reset bad settings like desktop boards.
